projects/components/src/select/select.component.scss

```diff
@@ -99,6 +104,10 @@
     display: flex;
     align-items: center;
 
+    &.open {
+      background-color: transparent;
```

Comment:
Is this doing anything or is it overridden by line 124 (indentation may be throwing me off here)

Reply:
done